source/configuration_manual/authentication/oauth2.rst

```diff
@@ -211,6 +211,14 @@ The following algorithms are supported
 There is currently no support for EdDSA algorithms.
 ES supports any curve supported by OpenSSL for this purpose.
 
+OpenID.Discovery
+****************
+
+.. versionadded:: v2.3.16
+
+Support for `RFC 7628 <https://datatracker.ietf.org/doc/html/rfc7628#section-3.2.2>`__  OpenID Discovery (OIDC) can be achieved with
```

Comment:
Other RFC urls in documentation are mostly to the handy tools.ietf.org site or the official rfc-editor.org. Does using datatracker bring some benefit here?

Reply:
not really, it was first url offered.

Reply:
also, tools.ietf.org redirects here.

Reply:
I had not noticed that change. Maybe we should change all tools.ietf.org URLs then, but that is already out of scope for this PR.